app/queue/export_service.py

```
@@ -87,17 +91,34 @@ def create_export(export_svc_data, org_id, inventory_config, operation_args={},
             )
             _handle_export_response(response, exportFormat, exportUUID)
             return False
+    except HTTPException as e:
+        return _handle_export_error(
+            e,
+            e.code,
+            export_service_endpoint,
+            exportUUID,
+            applicationName,
+            resourceUUID,
+            session,
+            request_headers,
+        )
     except Exception as e:
-        logger.error(e)
-        request_url = f"{export_service_endpoint}/app/export/v1/{exportUUID}/{applicationName}/{resourceUUID}/error"
-        response = session.post(
-            url=request_url, headers=request_headers, data=json.dumps({"message": str(e), "error": 500})
+        return _handle_export_error(
+            e, 500, export_service_endpoint, exportUUID, applicationName, resourceUUID, session, request_headers
```

A few notes/questions for this line:
- Why the hardcoded 500 for the status code?
- It would be cleaner to just pass in the `request_url` instead of all of its pieces separately.
- It seems a bit odd to return `_handle_export_error()` when it's always gonna be False. Honestly instead of returning a boolean, I think it would make more sense for `create_export` to just raise an exception if it fails